jquery.dataTables/jquery.dataTables.d.ts

```diff
@@ -189,6 +189,17 @@ declare namespace DataTables {
         draw(reset?: boolean): DataTable;
 
         /*
+        * Look up a language token that was defined in the DataTables' language initialisation object.
+        *
+        * @param token The language token to lookup from the language object.
+        * @param def The default value to use if the DataTables initialisation has not specified a value.
+        * @param numeric If handling numeric output, the number to be presented should be given in this parameter. If not numeric operator is required (for example button label text) this parameter is not required.
+        *
+        * @returns Resulting internationalised string.
+        */
+        i18n(token: string, def: Object | string, numeric?: number): string;
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html
> If you’re tempted to use the type Object, consider using any instead. There is currently no way in TypeScript to specify an object that is “not a primitive”.